controller/api/models.py

```diff
@@ -298,6 +298,24 @@ def _start_containers(self, to_add):
         if set([c.state for c in to_add]) != set(['up']):
             err = 'warning, some containers failed to start'
             log_event(self, err, logging.WARNING)
+        # if the user specified a health check, try checking to see if it's running
+        try:
```

Comment:
Should we add some sort of sleep/retry logic here? I could see apps that take a while to start up result in failure each time because the healthchecks don't respond quickly enough.